tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -5666,6 +5694,51 @@ <h2>Show similarity to selected datapoint</h2>
         }
       },
 
+      sortPdFeatures_: function() {
+        // First hide all existing PD plots before refreshing.
+        this.hideAllPdElements_();
+
+        const urlParams = {
+          example_index: this.globalPdPlots ? -1 : this.selected[0],
+          inference_address: this.inferenceAddress,
+          model_name: this.modelName,
+          model_type: this.modelType,
+          model_version: this.modelVersion,
+          model_signature: this.modelSignature,
+          use_predict: this.usePredictApi,
+          predict_output_tensor: this.predictOutputTensor,
+          predict_input_tensor: this.predictInputTensor,
+        };
+
+        this.isSortingEligibleFeatures = true;
+        // Call into the backend to sort the possible features to show plots for.
+        if (!this.local) {
+          const url =
+              tf_backend.addParams(
+                '/data/plugin/whatif/sort_eligible_features', urlParams);
+          const setEligibleFields = result => {
+            this.set('partialDepPlotEligibleFeatures', result.value);
+          };
+          const sortErrorCallback = () => {
+            this.isSortingEligibleFeatures = false;
+          }
+          this.makeAsyncRequest_(url,
+            setEligibleFields, null, 'plot ordering', sortErrorCallback.bind(this));
+        } else {
+          urlParams['features'] = this.partialDepPlotEligibleFeatures;
```

Comment:
nit: Not sure which is faster, but we could pass this to TensorBoard version as well so you don't have to call python get_eligible_features again. Not necessary.

Reply:
that part isn't the bottleneck of this operation, the generating of all PD plots for all features is. so i don't think its necessary.

Reply:
sounds good.